### ISSUE TITLE
Topic redirect to inexistent topic crashes SS

### ISSUE THREAD

**Issue by mariusursache** [silentrob/superscript#156]
Testing is simple: 
```
  + test missing topic
  - ^topicRedirect(supercalifragilisticexpialidocious,hello) Test OK.
```

**Comment by karovski** [comment]
I got this issue a lot of times so far:
```
superscript/lib/topics/topic.js:280
            Common.walkReplyParent(reply._id, function (err, replyThreads) {
```
It's most likely a consequence of the conversation implementations. Will try to check if it happens in cases other than redirecting to a non-existent topic.

**Comment by silentrob** [comment]
Fixed in 1c6ea3e
